rts/lra/lra-coordinator/pom.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jboss.narayana.rts</groupId>
+        <artifactId>lra-parent</artifactId>
+        <version>5.6.5.Final-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>lra-coordinator</artifactId>
+    <name>LRA Coordinator</name>
+    <packaging>war</packaging>
+
+    <properties>
+        <narayana.version>${project.version}</narayana.version>
+
+        <failOnMissingWebXml>false</failOnMissingWebXml>
+        <narayana.nodename.property>CoreEnvironmentBean.nodeIdentifier</narayana.nodename.property>
+        <swarm.http.port>8080</swarm.http.port>
+    </properties>
+
+    <build>
+        <finalName>lra-coordinator</finalName>
+        <plugins>
+          <plugin>
+            <inherited>false</inherited>
+            <groupId>org.sonatype.plugins</groupId>
+            <artifactId>nexus-staging-maven-plugin</artifactId>
+          </plugin>
+            <plugin>
+                <groupId>org.wildfly.swarm</groupId>
+                <artifactId>wildfly-swarm-plugin</artifactId>
+                <version>${version.wildfly-swarm}</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>package</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <environment>
+                        <swarm.http.port>${swarm.http.port}</swarm.http.port>
+                    </environment>
+                    <properties>
+                        <lra.http.port>${swarm.http.port}</lra.http.port>
+                    </properties>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <configuration>
+                    <skipITs>true</skipITs>
```

Comment:
with the release profile, is it really need to still skipping the tests? The arquillian is still not capable to resolve the artifacts?

Reply:
Running LRASmokeIT gives me the following shrinkwrap error:

```java
java.lang.RuntimeException: Could not invoke deployment method: public static org.jboss.shrinkwrap.api.Archive io.narayana.lra.test.LRASmokeIT.createDeployment() throws java.lang.Exception
        at io.narayana.lra.test.LRASmokeIT.createDeployment(LRASmokeIT.java:79)
```

Reply:
hm... strange, I need to check what's happening. I still do not understand why is that :/